src/features/device-state/state-get-utils.ts

```diff
@@ -39,8 +39,15 @@ export const setDefaultConfigVariables = (config: Dictionary<string>): void => {
 };
 
 export const getConfig = (
-	device: AnyObject | undefined,
-	application: AnyObject = device?.belongs_to__application[0],
+	device:
```

Comment:
Theoretically this could be considered a breaking change as it is narrowing the types, however, it is narrowing to a more correct typing (e.g. if you were passing something that does not satisfy these typings, you would probably be running into runtime errors), so I decided to keep this in a patch

Reply:
Yeah, imo patch is fine for things like this where these properties were always required but were just not typed narrowly enough to cover the actual requirements. It's part of the progressive typing we have where `AnyObject` is purely better than `any` (guards against more mistakes) and so we use that to avoid potential "perfect being the enemy of good" issues but it's even more better to do this specific typing